src/aioamazondevices/api.py

```diff
@@ -857,72 +883,55 @@ async def get_devices_data(
     ) -> dict[str, AmazonDevice]:
         """Get Amazon devices data."""
         self._devices = {}
-        for key in URI_QUERIES:
-            _, raw_resp = await self._session_request(
-                method=HTTPMethod.GET,
-                url=f"https://alexa.amazon.{self._domain}{URI_QUERIES[key]}",
-            )
-            _LOGGER.debug("Response URL: %s", raw_resp.url)
-            response_code = raw_resp.status
-            _LOGGER.debug("Response code: |%s|", response_code)
+        _, raw_resp = await self._session_request(
+            method=HTTPMethod.GET,
+            url=f"https://alexa.amazon.{self._domain}{URI_DEVICES}",
+        )
+        _LOGGER.debug("Response URL: %s", raw_resp.url)
```

Comment:
```suggestion
```

I would rather add the logging in _session_request and remove it from all the other places.
Maybe a separate PR to optimize it before.